website/source/api/system/generate-root.html.md

```diff
@@ -63,6 +71,11 @@ generation attempt can take place at a time.
 - `pgp_key` `(string: <optional>)` – Specifies a base64-encoded PGP public key.
   The raw bytes of the token will be encrypted with this value before being
   returned to the final unseal key provider.
+- `otp` `(string: <optional>)` – Specifies a base64-encoded 16-byte
```

Comment:
This was purposefully removed. The docs always reflect the current version of Vault and we don't want people to provide their own OTPs anymore.